hexrd/ui/hexrd_config.py

```diff
@@ -699,8 +699,8 @@ def images_dict(self):
         return self.intensity_corrected_images_dict
 
     @property
-    def masked_images_dict(self):
-        """Get an images dict where masks have been applied"""
+    def masks_dict(self):
```

Comment:
Maybe we should rename this to `raw_masks_dict` to clarify that it only returns raw masks?